src/vs-bicep/Bicep.VSLanguageServerClient.Vsix/TextMateSyntaxes/bicep.tmTheme

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
This is a copy of theme file used in visual studio.

Reply:
What do you mean by copy?

Reply:
As discussed offline, it's the one that'll be used in VS by default